elements/lisk-p2p/src/peer/base.ts

```diff
@@ -245,9 +242,9 @@ export class Peer extends EventEmitter {
 
 		// This needs to be an arrow function so that it can be used as a listener.
 		this._handleRawMessage = (packet: unknown): void => {
-			let message;
+			const message = packet as P2PMessagePacket;
```

Comment:
`_handleRawRPC` has similar pattern, so it should be the same.
I think it would be better if the `packet` is unknown and after the validation the type is changed. ie)
```
this._handleRawMessage = (packet: unknown): void => {
    try {
		validateProtocolMessage(packet);
	} catch (error) {
		this.emit(EVENT_INVALID_MESSAGE_RECEIVED, {
			packet,
		.......................			
const message = packet as P2PMessagePacket;
...

```